webcompat/webhooks/helpers.py

```diff
@@ -78,3 +82,25 @@ def get_payload_signature(key, payload):
     '''Compute the payload signature given a key.'''
     mac = hmac.new(key, msg=payload, digestmod=hashlib.sha1)
     return mac.hexdigest()
+
+
+def extract_priority_label(body):
+    """Parse url from body and query the priority labels."""
+    host_name = domain_name(extract_url(body))
+
+    if host_name:
+        priorities = ['critical', 'important', 'normal']
+
+        # Find host_name in DB
+        for site in site_db.query(Site).filter_by(url=host_name):
+            return 'priority-{}'.format(priorities[site.priority-1])
+
+        # No host_name in DB, find less-level domain (>2)
+        # If host_name is lv4.lv3.example.com, find lv3.example.com/example.com
+        domains = host_name.split('.')
+        for i in list(reversed(range(2, len(domains)))):
+            shorten_domain_name = '.'.join(domains[(len(domains)-i):])
+            for site in site_db.query(Site).filter_by(url=shorten_domain_name):
+                return 'priority-{}'.format(priorities[site.priority-1])
```

Comment:
I do the 1-pass test today, with full host name matching, only 294/7840 issues found. Compared with 2-passes test before is 3212/7348.
For me I choose to keep 2-passes, and collect black list (manually or we can use alexa site info API to know which domain we should always not do 2-passes)